### PR TITLE
[NFC] relationals: Tidy up get_order_string

### PR DESCRIPTION
**Pull request by svenvh**
Remove a misleading comment at the end.  Improve comments in general, and use `true`/`false` for `bool`s.

### CODE REVIEW

**Review by bashbaug — APPROVED** [review]
Thank you for fixing these!  This will aid understanding the next time we need to modify these tests.

File: test_conformance/relationals/test_shuffles.cpp [hunk 1]
```diff
@@ -201,14 +201,13 @@ const char *get_order_string( ShuffleOrder &order, size_t vecSize, cl_uint lengt
 
     size_t j, idx;
 
-    // Assume we don't have to use numbers
-    byNumber = 0;
-    // Check to see
+    // Assume we don't have to use numbered indices (.s0123...).
+    byNumber = false;
```
Comment (EwanC) [inline test_conformance/relationals/test_shuffles.cpp:205]: Can we remove the `byNumber` pass-by-value parameter of `get_order_string` if we always set the value to `false` at the start of the function? And just have `bool get_order_string = false;` here instead.

File: test_conformance/relationals/test_shuffles.cpp [hunk 2]
```diff
@@ -201,14 +201,13 @@ const char *get_order_string( ShuffleOrder &order, size_t vecSize, cl_uint lengt
 
     size_t j, idx;
```
Comment (EwanC) [inline test_conformance/relationals/test_shuffles.cpp:202]: Would be nice if we could move these declarations down into the usages

**Review by EwanC — APPROVED** [review]
LGTM, some minor suggestions it's fine to pass on if it's out of scope for this PR

**Comment by svenvh** [comment]
> LGTM, some minor suggestions it's fine to pass on if it's out of scope for this PR

Thanks! There is indeed more to improve (I noticed the strange use of the pass-by-value parameter too), but I was trying to keep this PR mainly focused around the remark [here](https://github.com/KhronosGroup/OpenCL-CTS/pull/1567#discussion_r1031454226).

**Pull request merged**